users/decorator.py

```diff
@@ -0,0 +1,35 @@
+import jwt
+import json
+
+from django.http import (
+    JsonResponse,
+    HttpResponse
+)
+
+from users.models import User
+from dr_martens.settings import (
+    SECRET_KEY,
+    ALGORITHM
+)
+
+def login_check(func):
+    def wrapper(self, request, *args, **kwargs):
+        access_token = request.headers.get("Authorization", None)
+        if access_token is not None:
+            try:
+                decode_token  = jwt.decode(access_token, SECRET_KEY, algorithms = ALGORITHM)
+                users_id      = decode_token['users_id']
+                users         = User.objects.get(id = users_id)
+                request.users = users
+                return func(self, request, *args, **kwargs)
+
+            except jwt.DecodeError:
+                return JsonResponse({'message' : 'INVALID_USER'}, status = 400)
```

Comment:
```suggestion
                return JsonResponse({'message' : 'INVALID_TOKEN'}, status = 400)
```
잘못된 인증정보 제공상황이므로 유저 식별이 아니라 토큰 자체가 디코드에러 상황이니까 변경해주세요.